tests/test_freshdesk_bookmarks.py

```diff
@@ -143,10 +142,27 @@ def test_run(self):
         for stream in self.test_streams:  # Add supported streams 1 by 1
             with self.subTest(stream=stream):
 
+                # Assert failures for streams not present in first sync (loss of coverage)
+                if stream in self.first_sync_empty:
+                    self.assertTrue(False, msg="Stream: {} no longer in sync 1. Check test data".format(stream))
+
+                    continue
+
                 # Assert failures for streams present in first sync but not second sync
                 if stream in self.second_sync_empty:
-                    print("Commented out failing test case. TODO add JIRA ID. Stream: {}".format(stream))
-                    #self.assertTrue(False, msg="Stream: {} present in sync 1, missing in sync 2!".format(stream))
+                    if stream == 'conversations':
+                        print("Commented out failing test case. TODO add JIRA ID. Stream: {}".format(stream))
+                        # conversations is a child of tickets. When the child object (conversation / note)
+                        # is updated, the parent object (ticket) is also updated.  The ticket is then being
+                        # sync'd after update but the child conversation of that updated ticket is not
+                        # (at least when the actual note text was updated.  There are several ways to update
+                        # the child).
+                        #self.assertTrue(False, msg="Stream: {} present in sync 1, missing in sync 2!".format(stream))
```

Comment:
Can you write up the Jira ticket and link it before you merge this in. The template for writing up a Bug can be found here https://jira.talendforge.org/browse/TDL-16092 if anything is unclear we can do this together.

Reply:
Done